test/api/unit/models/build.test.js

```diff
@@ -1,5 +1,6 @@
 const { expect } = require('chai');
 const { stub } = require('sinon');
+const _ = require('underscore');
```

Comment:
hmm, i can't see where this is used...

Reply:
removed .... we were trying to use the wait functionality and opted out